tests/test_code_analyzer.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+import threading
+
+from bionic.code_analyzer import CodeHasher, CodeHashPrefix
+
+
+def test_code_hasher():
+    def barray(value):
+        return bytearray(value, "utf8")
+
+    circular_dict = {"k1": "v1"}
+    circular_dict["k2"] = circular_dict
+
+    def f1():
+        v = 10
```

Comment:
Is this testing something different from just writing `return 10`?

Reply:
I think I lost direct return method when refactoring test. Added it back to check that returning directly vs returning a variable are treated differently.